app/gazebo/tug/plugins/tuginterface/interface/CMakeLists.txt

```diff
@@ -6,14 +6,14 @@ set(gazebo_assistiverehab_tuginterface_source src/tuginterface.cpp src/tugserver
 set(gazebo_assistiverehab_tuginterface_headers include/tuginterface.h include/tugserver.h include/utils.h)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR}
+                    ${TIRPC_HEADERS}
                     ${Boost_INCLUDE_DIRS}
                     ${GAZEBO_INCLUDE_DIRS})
 link_directories(${GAZEBO_LIBRARY_DIRS})
```

Comment:
Is this really required? Asking 'cause `link_directories` is quite old-fashioned cmake stuff.

Reply:
I found it in the tutorial [here](http://gazebosim.org/tutorials?tut=plugins_hello_world). 
I think that linking to `${GAZEBO_LIBRARIES}` through `target_link_libraries()` is sufficient and we can remove the `link_directories`.

Reply:
Just double check it to make sure that this is sufficient.

Reply:
Yes I double checked and it's sufficient.